src/game/AI/ScriptDevAI/scripts/outland/hellfire_citadel/shattered_halls/shattered_halls.h

```diff
@@ -72,8 +72,37 @@ enum
     NPC_HEARTHEN_GUARD          = 17621,
     NPC_SHARPSHOOTER_GUARD      = 17622,
     NPC_REAVER_GUARD            = 17623,
-
-    WORLD_STATE_CUSTOM_SPAWN_WAVES = 5400004,
+    
+    // First Group in Dungeon should not drop any loot/give any rep
+    NPC_SHATTERED_HAND_HEATHEN      = 17420,
+    NPC_SHATTERED_HAND_SAVAGE       = 16523,
+    STRING_ID_ENTRANCE_GROUP        = 5400002,
+    
+    // Used for Hall of Fathers intro
+    WORLD_STATE_LEGIONNAIRE_001     = 5400001,              // Prevents that shattered hand savage/heathen can respawn when legionnaire 01 is dead
```

Comment:
These world state vars are meant to be defined in file WorldStateDefines.h - the reason is to have one location where you can look and find all of them. Otherwise its confusing to find them all.